assets/scripts/src/choices.js

```diff
@@ -282,9 +281,9 @@ class Choices {
     // Reinstate passed element
     this.passedElement.classList.remove(this.config.classNames.input, this.config.classNames.hiddenState);
     this.passedElement.removeAttribute('tabindex');
-    this.passedElement.removeAttribute('style', 'display:none;');
+    this.passedElement.removeAttribute('style');
```

Comment:
Will this not remove all styles? I can't remember off the top of my head whether we had more!